legal-api/src/legal_api/reports/report.py

```diff
@@ -476,6 +474,9 @@ def _format_name_change_data(self, filing):
         if self._filing.filing_type == 'alteration':
             from_legal_name = meta_data.get('alteration', {}).get('fromLegalName')
             to_legal_name = meta_data.get('alteration', {}).get('toLegalName')
+        if self._filing.filing_type == 'correction':
+            from_legal_name = meta_data.get('correction', {}).get('fromLegalName')
```

Comment:
Missing data for changeOfName for corrections:

![image](https://github.com/bcgov/lear/assets/3484109/db55cf81-0228-4a51-a5e8-a1379e815b94)